addons/web/static/src/search/search_model.js

```diff
@@ -38,7 +38,7 @@ const { DateTime } = luxon;
  * @property {Context} context
  * @property {DomainListRepr} domain
  * @property {string[]} groupBy
- * @property {string[]} orderBy
+ * @property {OrderTerm[]} orderBy
```

Comment:
would be nice to import description of the type OrderTerm like in

/** @typedef {import("./error_service").UncaughtError} UncaughtError */